src/decorator/options/ColumnOptions.ts

```diff
@@ -67,7 +68,15 @@ export interface ColumnOptions extends ColumnCommonOptions {
     /**
      * Default database value.
      */
-    default?: any
```

Comment:
I'm not sure if it won't be a breaking change. I'm not sure what value users could use in this place (Date? Buffer? Anything else that was just stringified and was working?).